macro/src/lib.rs

```diff
@@ -246,7 +243,12 @@ fn fetch_metadata(args: &RuntimeMetadataArgs) -> Result<subxt_codegen::Metadata,
                 .and_then(|b| subxt_codegen::Metadata::decode(&mut &*b).map_err(Into::into))
                 .map_err(|e| CodegenError::from(e).into_compile_error())?
         }
+        #[cfg(feature = "jsonrpsee")]
```

Comment:
fetch metadata via `runtime_metadata_insecure_url` is only possible with the `feature = "jsonrpsee` now to enable opting out from jsonrpsee/tokio